charts/fact-admin/values.yaml

```diff
@@ -26,5 +26,6 @@ nodejs:
     REDIS_HOST: "fact-admin-session-storage-{{ .Values.global.environment }}.redis.cache.windows.net"
     FRONTEND_URL: "https://fact.{{ .Values.global.environment }}.platform.hmcts.net"
     IMAGE_BASE_URL: "https://fact{{ .Values.global.environment }}.blob.core.windows.net/images"
+    IDAM_USER_DASHBOARD: "https://idam-user-dashboard.{{ .Values.global.environment }}.platform.hmcts.net"
```

Comment:
We probably need to remove the other variables that won't be used after this is merged in 

For example: IDAM_ADD_USER_URL won't be used afterwards

Reply:
IDAM_UPDATE_USER_URL, IDAM_ADD_USER_URL (think its just those two)

Reply:
I'd also change the variable to be IDAM_USER_DASHBOARD_URL to denote the purpose of it and to follow the same structure as the other variable names above

Reply:
fixed, added cnp flux config line here https://github.com/hmcts/cnp-flux-config/pull/16901